src/main/Liquor.java

```diff
@@ -0,0 +1,18 @@
+package main;
+
+public class Liquor implements Visitable {
+    private final double price;
+
+    public Liquor(double item) {
```

Comment:
Constructor parameter names should (in general) match the field name. When you need to do this, you need to qualify the assignment with the `this` object reference, i.e `this.price = price`, otherwise Java thinks you're referring to the parameter when it resolves the name.